src/buildercore/checks.py

```diff
@@ -2,14 +2,30 @@
 
 these should compliment not replicate any project configuration validation."""
 
-from . import project
+from . import core, project
 from .project import repo
 
+class AccessProblem(RuntimeError):
+    pass
+
+class StackAlreadyExistingProblem(RuntimeError):
```

Comment:
`StackAlreadyExistsProblem` or `ExtantStackProblem` or perhaps just `StackExistsError` might be better